freqtrade/exchange/exchange.py

```diff
@@ -138,7 +139,7 @@ def _init_ccxt(self, exchange_config: dict, ccxt_module=ccxt,
         # Find matching class for the given exchange name
         name = exchange_config['name']
 
-        if name not in ccxt_module.exchanges:
+        if not is_exchange_supported(name, ccxt_module):
```

Comment:
I don't think passing in ccxt_module makes too much sense. 

i see that it's been there before - however ccxt supports the same exchanges in sync and async mode - so it either fails for both - or succeeds for both possible types of ccxt_module

Reply:
Yeah, but taking ccxt.exchanges in the init of async part that utilizes ccxt_async instead of ccxt instance seems for me unsymmetric so I did it (left as it was, only reformulated) on this way only for this reason. Theoretically ccxt and ccxt_async may return different lists.